packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.tsx

```diff
@@ -58,6 +58,7 @@ export type KibanaPageTemplateSolutionNavProps = Omit<
    */
   isOpenOnDesktop?: boolean;
   onCollapse?: () => void;
+  canBeCollapsed?: boolean;
```

Comment:
I like props comments for increasing understanding and for future devs touching this file.
```suggestion
  /**
   * Allows hiding of the navigation by the user.
   * If false, forces all breakpoint versions into the open state without the ability to hide.
   */
  canBeCollapsed?: boolean;
```